libraries/networking/src/udt/SendQueue.h

```diff
@@ -42,21 +42,8 @@ class SendQueue : public QObject {
     Q_OBJECT
     
 public:
-    
-    class DoubleLock {
-    public:
-        DoubleLock(std::mutex& mutex1, std::mutex& mutex2) : _mutex1(mutex1), _mutex2(mutex2) { lock(); }
-        ~DoubleLock() { unlock(); }
-        
-        DoubleLock(const DoubleLock&) = delete;
-        DoubleLock& operator=(const DoubleLock&) = delete;
-        
-        void lock() { std::lock(_mutex1, _mutex2); }
-        void unlock() { _mutex1.unlock(); _mutex2.unlock(); }
-    private:
-        std::mutex& _mutex1;
-        std::mutex& _mutex2;
-    };
+    using clock = std::chrono::high_resolution_clock;
```

Comment:
Oh - ignore my earlier comment with this. 
I'm not a huge fan of this, makes it less clear what we're using for timing - is `high_resolution_clock` too long? If so I would prefer an alias that gives a bit more context - something like `high_res_clock`.